packages/nimble-components/src/breadcrumb-item/styles.ts

```diff
@@ -1,81 +1,59 @@
 import { css } from '@microsoft/fast-element';
 import { display } from '../utilities/style/display';
 import {
-    bodyFontColor,
-    borderHoverColor,
-    borderWidth,
     controlHeight,
     iconSize,
-    linkFont,
+    linkDisabledFontColor,
+    mediumPadding,
     placeholderFontColor
 } from '../theme-provider/design-tokens';
-import { focusVisible } from '../utilities/style/focus';
+import { styles as anchorStyles } from '../patterns/anchor/styles';
 
 export const styles = css`
-    ${display('inline-flex')}
+    ${anchorStyles}
 
-    :host {
-        height: ${controlHeight};
-        padding-left: calc(4px - ${borderWidth});
+    @layer base {
+        ${display('inline-flex')}
 
-        ${
-            /* When href removed the .control element is also removed
-             so this becomes the fallback color for the slot */ ''
-        }
-        color: ${bodyFontColor};
-        font: ${linkFont};
-    }
-
-    .listitem {
-        display: flex;
-        align-items: center;
-    }
-
-    .control {
-        color: var(--ni-private-breadcrumb-link-font-color);
-        display: flex;
-        align-items: center;
-        justify-content: center;
-        border: ${borderWidth} solid transparent;
-        padding-right: calc(4px - ${borderWidth});
-        text-decoration: none;
-    }
+        :host {
+            height: ${controlHeight};
+            padding-left: ${mediumPadding};
 
-    .control:hover {
-        text-decoration: underline;
-    }
-
-    .control${focusVisible} {
-        border: ${borderWidth} solid ${borderHoverColor};
-        outline: 2px solid ${borderHoverColor};
-        outline-offset: 1px;
-    }
-
-    .control:active {
-        color: var(--ni-private-breadcrumb-link-active-font-color);
-        text-decoration: none;
-    }
+            ${
+                /* When href removed the .control element is also removed
+                so this becomes the fallback color for the slot */ ''
+            }
+            color: ${linkDisabledFontColor};
+        }
 
-    [part='start'] {
-        display: none;
-    }
+        .listitem {
+            display: flex;
+            align-items: center;
+        }
 
-    .content {
-        pointer-events: none;
-    }
+        .control {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            margin-right: ${mediumPadding};
+            text-decoration: none;
+        }
 
-    [part='end'] {
-        display: none;
-    }
+        :host(:not([href])) slot:not([name]),
```

Comment:
From our [CSS guidelines](https://github.com/ni/nimble/blob/main/packages/nimble-components/docs/css-guidelines.md#prefer-cascade-for-state-changes)
>If you find yourself in complex logic with lots of :not() selectors it's possible the code should be reorganized to leverage the CSS cascade for overriding states.

I'm not saying this is overly complex, but it's still preferable to avoid `not` if practical. Could you take a stab at using the cascade instead or comment what makes that difficult?

Reply:
There are two kinds of `:not()` usages here:
1. `slot:not([name])` - Used to style the default slot. Since breadcrumb item also has `start` and `end` slots, we have to disambiguate, and this is the most direct way to select just the default slot. 
2. `:host(:not([href]))` - Used to select when the breadcrumb item has no `href` set. Again, this is the most direct way to represent that.

If we use the cascade, we end up with more lines of CSS, which are more roundabout and harder to understand:
```css
slot {
    display: block;
    margin-right: ${mediumPadding};
}

:host([href]) slot,
slot[name="start"],
slot[name="end"] {
    display: unset;
    margin-right: unset;
}

:host([href='']) slot {
    display: block;
    margin-right: ${mediumPadding};
}
```
We can't combine the first and third rule blocks, because `:host([href]) slot` and `:host([href='']) slot` have the same specificity. That means order dictates precedence, so we have to make sure the `:host([href='']) slot` rule comes later. The fact we have to care about specificities (and I had to use an online calculator to check them) is another argument that this is more complex.

If I'm being a dope and there's a simpler way to write this, or there's an argument that the version without `:not()` is better, I'm open to hearing it. I also wish I could use `:not()` without spending extra time justifying it, but I guess that's a necessary evil.

Reply:
Some duplication is pretty much expected with the style conventions. We group by DOM order, not by the sets of configured properties, so start and end selectors get split up and often (not always) have some duplicated styles.

The complexity the style guide rules push to optimize for is picking an element in the DOM / template, jumping to the CSS, and seeing exactly what styles are expected on that element given the state of the page.

Stuff like the `slot:not([name])` makes that hard because one potentially has to jump to multiple styles to understand what applies to an element. By that understanding of complexity, low complexity is seeing `slot[name="start"]` applies to the "start slot" and high complexity is having to understand `slot:not([name])` is used to style the "start slot".

My gut tells me we can align with our CSS guidelines better. I'll take a stab at looking at the styles in a separate branch but fyi might be a couple days (don't think there is a specific urgency here). May be a good way to capture additional style guideline clarifications as well.

Reply:
> high complexity is having to understand `slot:not([name])` is used to style the "start slot".

Actually I think I even messed that up. `slot:not([name])` is to target a slot without a name, so the default slot 😂

Reply:
Yeah, I figured you meant "default slot". 😊

> We group by DOM order, not by the sets of configured properties

Good point. I hadn't followed that convention in my earlier reply, though since we're setting `display: none` on the `start` and `end` parts (in the anchor styles), we can leave out the styling for those slots. But there _is_ a `separator` slot I hadn't accounted for.

So if we avoid `:not()`, we actually end up with this:

```css
/* Since this does not refer to a specific element in the template, I guess it should go where the first instance of this element type occurs? */
slot { 
    display: block;
    margin-right: ${mediumPadding};
}

:host([href]) slot {
    display: unset;
    margin-right: unset;
}

:host([href='']) slot {
    display: block;
    margin-right: ${mediumPadding};
}

slot[name='separator'] {
    ...
    margin-right: unset;
}
```

For comparison, using `:not()`, we have:

```css
:host(:not([href])) slot:not([name]),
:host([href='']) slot:not([name]) {
    display: block;
    margin-right: ${mediumPadding};
}
/* no change needed to separator slot styling */
```

> The complexity the style guide rules push to optimize for is picking an element in the DOM / template, jumping to the CSS, and seeing exactly what styles are expected on that element given the state of the page. Stuff like the `slot:not([name])` makes that hard because one potentially has to jump to multiple styles to understand what applies to an element.

True, but importing shared styles (from the patterns directory) has that problem to an even greater extent, since you have to scan the entirety of another file. It's obviously a trade-off.

I agree that using `:not()` _can_ lead to the kind of complexity you're talking about, but in this particular case, since `slot:not([name])` refers to exactly one element in the template, it's not actually any more complex than a selector like `slot[name="start"]` (for example). If it's not obvious that `slot:not([name])` means "default slot", we could alway s add a comment.

Reply:
It turns out all of this was essentially moot, because I can just move the padding to the separator rather than conditionally adding left/right padding to the link (or text) elements. 🤦‍♂️